cmd/wallet-sdk-js/jsinterop/indexeddb/indexeddb.go

```diff
@@ -0,0 +1,658 @@
+//go:build js && wasm
+
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package indexeddb
+
+//nolint:gci
+import (
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+	"syscall/js"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/spi/storage"
+)
+
+// TODO (#2528): Proper implementation of all methods.
```

Comment:
I think you should remove the extra TODOs and methods that aren't applicable here (there's a bunch of things done here for aries-specific things). (At a minimum, the TODOs should be removed at least to avoid confusion in the future)

Reply:
Storage will be injectable from JS. And will be implemented in JS too.

Reply:
You should still at least get rid of the TODOs that don't make sense in this project, and any unused public methods. It's better to not have it in the commit history.